src/containers/SearchPage/SearchPageWithMap.js

```diff
@@ -321,6 +321,24 @@ export class SearchPageComponent extends Component {
       ...customSecondaryFilters,
     ];
 
+    // Inject this temporary long filter for testing
```

Comment:
Inject through configListing:
- https://github.com/sharetribe/web-template/blob/main/src/config/configListing.js#L49
- https://github.com/sharetribe/web-template/blob/main/src/util/configHelpers.js#L904

Then check that the min & max are valid
https://github.com/sharetribe/web-template/blob/main/src/util/configHelpers.js#L607
Something like this:
```js
            : name === 'min'
            ? validMin(value, schemaType)
            : name === 'max'
            ? validMax(value, schemaType, data)
```

They should be 
- Integers ([Number.isInteger()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isInteger))
- Between [MIN_SAFE_INTEGER](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MIN_SAFE_INTEGER) and [MAX_SAFE_INTEGER](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER), and
- Max should be bigger than min (which is why I added data aka config to validMax in the example)

And the validation check should be added for listing fields as well as user fields